spanner/api/Spanner.Samples.Tests/CreateInstanceWithProcessingUnitsTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Grpc.Core;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class CreateInstanceWithProcessingUnitsTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public CreateInstanceWithProcessingUnitsTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public void TestCreateInstanceWithProcessingUnits()
+    {
+        var createInstanceWithProcessingUnitsSample = new CreateInstanceWithProcessingUnitsSample();
+        var instance = createInstanceWithProcessingUnitsSample.CreateInstance(_spannerFixture.ProjectId, _spannerFixture.InstanceId));
```

Comment:
This InstanceId was already present on the fixture so I guess it is being used for something. It's probably the instance that has already been created and that is used for everything else. If that's the case this test will always fail.

Reply:
Good catch :) Thanks.